gateway/mw_request_size_limit.go

```diff
@@ -64,6 +68,18 @@ func (t *RequestSizeLimitMiddleware) ProcessRequest(w http.ResponseWriter, r *ht
 
 	vInfo, _ := t.Spec.Version(r)
 
+	globalConfigMaxRequestBodySize := t.Spec.GlobalConfig.HttpServerOptions.MaxRequestBodySize
+	logger.Debug("Global config max request body size limit is: ", globalConfigMaxRequestBodySize)
+	// Check global config max request body size first
+	if globalConfigMaxRequestBodySize > 0 {
+		logger.Debug("Checking global max request body size limit")
+		err, code := t.checkRequestLimit(r, globalConfigMaxRequestBodySize)
+		// If not OK, block
+		if code != http.StatusOK {
+			return err, code
+		}
+	}
+
 	logger.Debug("Global limit is: ", vInfo.GlobalSizeLimit)
 	// Manage global headers first
 	if vInfo.GlobalSizeLimit > 0 {
```

Comment:
What is vInfo.GlobalSizeLimit ?

Reply:
API version level size limit, instead of path level